### ISSUE TITLE
Responsiveness of map

### ISSUE THREAD

**Issue by diplomod** [diplomod/nordryggstien.no#68]
The map is integrated on http://dokuhl.de/nordryggen/
It does not fit perfectly on very large screens (see screenshot) and does not show at all on iPhone and iPad.
Can you please have a look?
Consult with @Immoz when questions occur.

![screen shot 2015-03-10 at 15 19 10](https://cloud.githubusercontent.com/assets/7487883/6576666/e7780ed8-c738-11e4-99b7-6f3a94c987f9.png)

**Comment by bokiftw** [comment]
I am quite sure that you have told me once, that you don't want map to show under 800px of width. So I made it like that. That's why this isn't shown under 800px (mobile and tablet). Otherwise I definitely won't put 800pc breakpoint.

Anyways this can be fixed but it will take some time. I will need to contact backend guys to discuss with them how this should be done now.

**Comment by diplomod** [comment]
Please contact backend guys - I cannot recall having specified 800 px.

Actually I always wanted to make sure that it fits all screens and fills the entire screen (anything else is not good for usability!)

Thanks - you'll sort it out 
@Immoz FYI - pease help as good as you can

**Comment by bokiftw** [comment]
This is finished. Just need to fix a little bug with the sticky red button. Hopefully will finish that tomorrow like all other stuff.

**Comment by diplomod** [comment]
The map still does not fit my 15,4-inch (1440 x 900) laptop screen. It shoud use the entire screen, but instead I see the community below.

![screen shot 2015-04-25 at 21 13 10](https://cloud.githubusercontent.com/assets/7487883/7334305/21f3013c-eb90-11e4-97e0-7f424a10604c.png)

**Comment by diplomod** [comment]
Fast: Wenn ich das Menü zuklappe, nutz die Map nicht den ganzen Bildschirm, sondern es wird rchts das Hintergrundbild angezeigt. Ist ja nicht im Sinne der Erfindung.
![screen shot 2015-04-30 at 17 10 24](https://cloud.githubusercontent.com/assets/7487883/7415898/e9c264f4-ef5b-11e4-8efe-726d015b688e.png)

**Comment by diplomod** [comment]
guys, works very smooth on tablet now. Just the latest comment is relevant
@lukassiedel bitte anschauen

**Comment by philipdomann** [comment]
and smartphone is also relevant: hans said he thinks that a few elements only have to disappear, like the "mouse wheel zoom" - button and the button "Vis Nordryggstien..".

**Comment by diplomod** [comment]
Ja. Und auf kleinen Schirmen haut das mit der Preview nicht immer hin:
![screen shot 2015-05-10 at 09 58 58](https://cloud.githubusercontent.com/assets/7487883/7553474/75b5df1c-f6fb-11e4-842d-69c30a0c48f5.png)
Die wird teilweise vom Menü überlagert
